qa/workunits/cephadm/nvme_initiator.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -ex
+
+sudo modprobe nvme-fabrics
+sudo modprobe nvme-tcp
+
+# import NVMEOF_GATEWAY_IP_ADDRESS and NVMEOF_GATEWAY_NAME
+source /etc/ceph/nvmeof.env
+
+GATEWAY=$(echo $NVMEOF_GATEWAY_NAME)
+IP=$(echo $NVMEOF_GATEWAY_IP_ADDRESS)
```

Comment:
Does this need to be complex?
```suggestion
GATEWAY="$NVMEOF_GATEWAY_NAME"
IP="$NVMEOF_GATEWAY_IP_ADDRESS"
```

Reply:
... or just use `NVMEOF_GATEWAY_NAME` and `NVMEOF_GATEWAY_IP_ADDRESS` directly?